jobs/aws/eks-distro-build-tooling/eks-distro-base-periodics.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+periodics:
+- name: eks-distro-base-periodic
+  labels:
+    image-build: "true"
+  # Runs every weekday (M-F) at 10am PST
+  cron: "* 18 * * 1-5"
+  cluster: "prow-postsubmits-cluster"
+  decoration_config:
+    gcs_configuration:
+      bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
+      path_strategy: explicit
+    s3_credentials_secret: s3-credentials
+  extra_refs:
+  - org: eks-distro-pr-bot
+    repo: eks-distro-build-tooling
+    base_ref: main
+  - org: eks-distro-pr-bot
+    repo: eks-distro
+    base_ref: main
+  - org: eks-distro-pr-bot
+    repo: eks-distro-prow-jobs
+    base_ref: main
+  spec:
+    serviceaccountName: periodics-build-account
+    containers:
+    - name: build-container
+      image: 316434458148.dkr.ecr.us-west-2.amazonaws.com/eks-distro/base:26f234d9da8bc4423bacb539caaece931808d28b
+      command:
+      - bash
+      - -c
+      - >
+        ./eks-distro-base/check_and_update.sh
+        &&
+        touch /status/done;
+      volumeMounts:
+      - name: ssh-auth
+        mountPath: /secrets/ssh-secrets
+        readOnly: true
+      - name: github-auth
+        mountPath: /secrets/github-secrets
+        readOnly: true
+      livenessProbe:
+        exec:
+          command:
+          - bash
+          - -c
+          - date +%s > /status/pending
+        periodSeconds: 10
+    - name: buildkitd
+      image: moby/buildkit:master-rootless
+      command:
+      - sh
+      args:
+      - /script/entrypoint.sh
+      volumeMounts:
+      - name: ssh-auth
+        mountPath: /secrets/ssh-secrets
+        readOnly: true
+      - name: github-auth
+        mountPath: /secrets/github-secrets
+        readOnly: true
+      livenessProbe:
+        exec:
+          command:
+          - sh
+          - -c
+          - test $(($(date +%s) - 15)) -lt $(cat /status/pending)
+        periodSeconds: 15
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+    volumes:
+    - name: ssh-auth
+      secret:
+        defaultMode: 256
+        secretName: ssh-secret  
```

Comment:
Same here, we probably want to mount a new ssh-secret as this is for a different account.